padl/transforms.py

```diff
@@ -1552,16 +1644,36 @@ def _pd_get_keys(transforms):
             deduped_keys.append(new_name)
         return deduped_keys
 
+    def _pd_get_non_target_stage_idx(self, stage_used):
+        return 1
+
+    def _pd_get_target_stage_idx(self, is_entire_transform=None):
+        return 0
+
+    def _pd_get_error_idx(self, stage: str):
+        """Track the index where a `Pipeline` fails from the one that fails on `self.pd_forward`.
```

Comment:
When referring to methods in docs please use 
```python
:meth:`Pipeline`
:meth:`self.pd_forward`
```
This allows sphinx to link to the methods in the docs.

Reply:
yeah, I am missing some of them